src/components/projects/ProjectsAPI.js

```diff
@@ -21,37 +21,9 @@ export default async function getProjects() {
 
 export async function getProject(id) {
   let handlerError = false;
+  const url = process.env.REACT_APP_PROJECTS_URL;
```

Comment:
let's construct the url, appending these to the env API HOST var:

const API_NAME = "projects"
const API_VERSION = "v1"

something like:
API_HOST/api/API_VERSION/API_NAME

Reply:
solved!